shoebot/grammar/nodebox.py

```diff
@@ -750,12 +751,15 @@ def align(self, align=LEFT):
         self._canvas.align = align
 
     # TODO: Set the framework to setup font options
-
     def fontoptions(
         self, hintstyle=None, hintmetrics=None, subpixelorder=None, antialias=None
     ):
         raise NotImplementedError(_("fontoptions() isn't implemented yet"))
 
+    def fontnames(self):
+        # from shoebot.util
+        return fontnames()
```

Comment:
I'm not sure this will work since it will call itself and die as there is no `self`.

You could import

`import shoebot.util as util`

Then call `util.fontnames`

Reply:
renamed the util function as list_pango_fonts for clarity, i think this is solved in 9a0e67360b65b68f4b2aebaef905148ca27e242e

Reply:
Please run Black on the source to format it (see underlines here):  
![image](https://user-images.githubusercontent.com/179677/111200170-5cf5f900-85b9-11eb-8004-59ff494591b9.png)

It looks like fontnames() needs updating to list_pango_fonts() at the bottom of the file too,

Reply:
Not sure if the comment         # description = face.describe()  is meant to still be there, if so it could do with a little more exposition.

Reply:
ok i think i addressed all these!